generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -9,16 +9,24 @@
     extra_params += " -mem-path /mnt/kvm_hugepage"
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+    s390x:
+        kvm_module_parameters = 'hpage=1'
     variants:
+        - 1M:
```

Comment:
If you wanna add hardware restrictions for different huge page settings, you should consider all platforms.
So I think you can just add `- 1M` variant and pick it at your test loop.

Reply:
I decided to add "only s390x" to avoid conflict with other architechture